update.xml

```diff
@@ -688,6 +688,17 @@
             <forum>https://forum.xda-developers.com/mi-a1/development/crdroid-5-0-pie-tissot-4-9-enforcing-t3882845</forum>
             <telegram>https://t.me/crdroid_tissot</telegram>
         </tissot>
+
+        <gemini>
+            <maintainer>M.H.K.JAHROMI</maintainer>
+            <devicename>MI 5</devicename>
+            <filename>crDroidAndroid-9.0-20190327-gemini-v5.2</filename>
+            <buildtype>Monthly</buildtype>
+            <download>https://drive.google.com/uc?export=download&confirm=-eED&id=1mTqYWU7ad0EsRdcUhivh2-xGssHzohCQ</download>
```

Comment:
XML doesn't accept "&" so please shortlink your urls (use bit.ly as described in How To)

Reply:
Okay